active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,385 @@
+| FirstButton | * Button displaying text and/or glyph indicating that the user may navigate to the first index. <br> * Automatically disabled when at first index. <br> * Can be set to not be visible when at the first index.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760409-485b8580-fc79-11e8-9545-a580f6016f6c.png) |
+| NextButton | * Button displaying text and/or glyph indicating that the user may navigate to the next index. <br> * Automatically disabled when at last index. <br> * Can be set to not be visible when at the last index. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760426-54474780-fc79-11e8-9446-ee2886941a24.png) |
+| PreviousButton | * Button displaying text and/or glyph indicating that the user may navigate to the previous index. <br> * Automatically disabled when at first index. <br> * Can be set to not be visible when at the first index.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760443-62956380-fc79-11e8-8f93-562a92fde666.png) |
+| Ellipsis | * Button, reading "...", used before or after the first/last index to indicate an accessible but omitted range of indexes. <br> * Visibility of the first/last index can be disabled. <br> <br> * Only visible when using button panel as the display mode.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760513-85277c80-fc79-11e8-926e-3453ea29b0a3.png) |
+| PrefixText | * Text displayed before the ComboBox or NumberBox indexing component. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760540-93759880-fc79-11e8-9c4f-3c3222b95a13.png) |
+| SuffixText | * When a total number of indices (N) is given, this suffix string will appear after the ComboBox indexing component or NumberBox component and read "of N". Localization will put "N" where it should be in a given language. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760557-9a9ca680-fc79-11e8-943f-3425d993fdeb.png) |
+
+# API Details
+
+```c++ 
+enum PagerDisplayMode
+{
+    Auto,
+    ComboBox,
+    NumberBox,
+    ButtonPanel,
+};
+
+enum PagerButtonVisibility
+{
+    Visible,
+    HiddenOnEdge,
+    Hidden,
+};
+
+runtimeclass PagerControlSelectedIndexChangedEventArgs
+{
+    Integer NewPageIndex{get; };
+    Integer PreviousPageIndex{get; };
+};
+
+runtimeclass PagerControl
+{
+    PagerControl();
+
+    PagerDisplayMode DisplayMode;
+    
+    Integer NumberOfPages;
+    
+    PagerButtonVisibility FirstButtonVisibility;
+    PagerButtonVisibility PreviousButtonVisibility;
+    PagerButtonVisibility NextButtonVisibility;
+    PagerButtonVisibility LastButtonVisibility;
+
+    FirstButtonCommand="FirstButtonPressedEvent"
+    PreviousButtonCommand="PreviousButtonPressedEvent"
+    NextButtonCommand="NextButtonPressedEvent"
+    LastButtonCommand="LastButtonPressedEvent"
+    PagerInputCommand="PagerInputEvent"
```

Comment:
I assume, with this section, you are trying to point which events on which button should execute that specific command right? In that case, maybe having a comment behind the commands is better to convey that, this might be a bit confusing to read.

Reply:
Yes Mike pointed this out as well so I will remove it from the API list.